lib/standard_deviation_module.rb

```diff
@@ -0,0 +1,29 @@
+module StandardDeviation
+
+  def standard_deviation_age
+
+    average = ages.sum.to_f / ages.length
+
+    subtracted = ages.map do |age|
+      age - average
+    end
+
+    rounded_numbers = subtracted.map do |number|
+      number.round(2)
+    end
+
+    squared_numbers = rounded_numbers.map do |number|
+      number ** 2
+    end
+
+    rounded_square_numbers = squared_numbers.map do |number|
+      number.round(2)
+    end
```

Comment:
You should make each one of these enumerables a separate method inside this module, and then call them inside this method (yes, a module can call its own methods!)

Reply:
Abstraction! Possible thing for us to do together here.